docs/src/layouts/HeroContainer.tsx

```diff
@@ -89,76 +73,31 @@ export default function HeroContainer(props: HeroContainerProps) {
       {right}
     </Box>
   );
-  if (disableMobileHidden) {
-    return (
-      <Box sx={{ overflow: 'hidden' }}>
-        <Container
-          sx={{
-            minHeight: { xs: 'auto', sm: 500 },
-            height: { md: 'calc(100vh - 120px)' },
-            maxHeight: { md: 700, xl: 850 },
-            transition: '0.3s',
-          }}
-        >
-          <Grid container sx={{ alignItems: 'center', height: '100%', mx: 'auto' }}>
-            <Grid
-              item
-              xs={12}
-              md={7}
-              lg={6}
-              sx={{
-                display: { xs: 'flex', md: 'block' },
-                minHeight: { xs: 500, sm: 700, md: 'initial' },
-                m: 'auto',
-                '& > *': {
-                  m: { xs: 'auto', md: 'initial' },
-                },
-              }}
-            >
-              {left}
-            </Grid>
-            <Grid item xs={12} md={5} lg={6} sx={{ maxHeight: '100%' }}>
-              {renderRightWrapper({
-                height: {
-                  xs: 'initial',
-                  md: 'calc(100vh - 120px)',
-                },
-                borderLeftWidth: { xs: 0, md: 1 },
-                borderBottomLeftRadius: { xs: 0, md: 12 },
-                mx: { xs: -2, sm: -3, md: 'initial' },
-              })}
-            </Grid>
-          </Grid>
-        </Container>
-      </Box>
-    );
-  }
+
+  const containerSx = {
+    minHeight: { xs: 'auto', sm: 500 },
+    height: { md: 'calc(100vh - 120px)' },
+    maxHeight: { md: 700, xl: 850 },
+    transition: '0.3s',
+  };
 
   return (
     <Box sx={{ overflow: 'hidden' }}>
-      <Container
-        sx={{
-          pt: { xs: 8, md: 0 },
```

Comment:
This isn't present in `containerSx` which is only when no `disableMobileHidden`.